### ISSUE TITLE
Don't understand the implementation of alignment-id attribute values

### ISSUE THREAD

**Issue by nevenjovanovic** [proiel/proiel-treebank#2]
After reading the [PROIEL XML specification](https://proiel.github.io/handbook/developer/proielxml.html), I was under the impression that the `alignment-id` attribute holds an id value from the aligned file ("the alignment-id on div, sentence and token elements must be interpreted in relation to the alignment-id on the source element"). In the `latin-nt.xml` file I find information which suggests it is aligned with the `greek-nt.xml`:

```xml

<source id="latin-nt" language="lat" alignment-id="greek-nt">
(...)
<div alignment-id="40">
    <div alignment-id="1">
      <title>Matthew 1</title>
      <sentence id="12667" status="reviewed" presentation-after=" ">
        <token id="250021" form="liber" citation-part="MATT 1.1" lemma="liber" part-of-speech="Nb" morphology="-s---mn--i" head-id="851355" relation="xobj" presentation-after=" " alignment-id="266690">

```
But, in the `greek-nt.xml` I do **not** find divs with id "40" or "1". True, there is a token with id "266690". The `sentence` element, however, is missing its alignment-id attribute. Is there a reason for the inconsistency on the level of divs and sentences? Are there plans to correct it?

We have a (modern Croatian) translation of the NT which we're aligning with the PROIEL's Greek NT. We're starting with the level of sentences. Eventually, we would like to be able to use that alignment to retrieve the Latin sentences as well -- either directly, or through the Greek aligned text. For that, it would be useful to have the Latin aligned at the sentence level as well.

**Comment by mlj** [comment]
The missing `id` attribute on `div` elements is an unintended inconsistency. (There used to be a motivation for its absence but this is no longer the case.) I will try to put together a new release this week to fix that.

The omission of `alignment_id` on `sentence` is intentional but perhaps a bit silly: The thinking was that since sentence alignment could be inferred from token alignment, it would just add redundant information. One could of course also say that about the alignment of the `div`s as well, but these are at least always one-to-one whereas the sentences may by many-to-many. But perhaps it is better to make this consistent on all levels.

**Comment by mlj** [comment]
The master branch now contains XML files with correct IDs on all `div` elements.

I'm going to wait until we have fixed a few other things (e.g. issue #4) and decided on the sentence alignments before I make a new release.

**Comment by mlj** [comment]
This has been fixed in release 20180408.
